target/linux/ramips/dts/mt7628an_glinet_gl-mt300n-v2.dts

```diff
@@ -71,6 +71,13 @@
 			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
 		};
 	};
+
+        i2c: i2c {
```

Comment:
Please drop `i2c:` label, mt7628an.dtsi already has it for built-in one.

Reply:
Done

Reply:
Come to think of it, the built-in i2c also uses the same pins, so you don't need to use i2c-gpio, you can just enable the existing `&i2c` node. Could you test if it also works?

Reply:
Probably you are right. Aforementioned `i2c` node have property `status = "disabled"`, so probably it is sufficient to add some lines like this:                      
```                                                                                
&i2c {                                                                             
  status = "okay";                                                                 
};                                                                                 
```                                                                                
I looked in `mt7628an.dtsi` file and I not found mention about any gpio in i2c description in dts and dtsi files, but I not so familiar with DeviceTree files. I can't fugure out where described link between `gpio4` and `scl` line, and link between `gpio5` and `sda` line.                                                                    
Unfortunately, I have no spare device for testing purpose at this moment, my single device now in use.

Reply:
The description for i2c pins can be found in the mt7628 datasheet.

Reply:
I read 'MT7628' datasheet and found information about i2c configuration.
I build two version of firmware for 'GL-MT300N V2' and tested on real device.
### Patch_1 (my):
```
i2c {
	compatible = "i2c-gpio";

	sda-gpios = <&gpio 5 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
	scl-gpios = <&gpio 4 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
};
```

I install packages `kmod-i2c-gpio` and `i2c-tools` on device with patch_1 and found i2c device `/dev/i2c-0`, but `i2cdetect -y -a 0` found nothing.

### Patch_2 (your):
```
&i2c {                                                                             
	status = "okay";                                                                 
};  
```

I install packages `kmod-i2c-mt7628` and `i2c-tools` on device with patch_2 and found i2c device `/dev/i2c-0`, and `i2cdetect -y -a 0` found connected i2c device. 

So, your thoughts were right about patch and I fixed my PR accordingly. Thank you for your advice!